arangod/V8Server/v8-replicated-logs.cpp

```diff
@@ -87,6 +90,27 @@ static LogId UnwrapReplicatedLog(v8::Isolate* isolate,
       obj->GetInternalField(SLOT_CLASS)->Uint32Value(TRI_IGETC).ToChecked()};
 }
 
+namespace {
```

Comment:
I feel that this code rather belongs into `lib/V8/v8-conv.[cpp|h]`

Reply:
if its less bulky in the usage maybe have both?

Reply:
It's mostly different in that `TRI_ObjectToUInt64` returns `0` if it's not a number, while my code returns an error. I can achieve my main goals with both: Allow larger ints than 32bits, and don't introduce any security issues.

While I personally prefer errors rather than silent defaulting, its probably more JavaScript-y this way, and I don't think it's much of a problem here either way. So I chose to not add additional code but use the existing one.